web/src/main/java/org/perfrepo/web/rest/security/FrontApiAuthenticationFilter.java

```diff
@@ -0,0 +1,61 @@
+package org.perfrepo.web.rest.security;
+
+import org.perfrepo.web.util.MessageUtils;
+
+import javax.servlet.*;
+import javax.servlet.annotation.WebFilter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ *
+ * @author Jiri Grunwald (grunwjir@gmail.com)
+ */
+@WebFilter(filterName = "FrontApiAuthenticationFilter", urlPatterns = {"/rest/json/*"})
+public class FrontApiAuthenticationFilter implements Filter {
```

Comment:
Probably worth renaming, since the same Filter will be used also for "non-front-end" access like pure REST, right?